### PR TITLE
Add private IP config to Forseti enforcer VM

### PR DESCRIPTION
**Pull request by ocervell**
This adds capability to have a private Forseti enforcer VM and makes it the default.

### CODE REVIEW

File: examples/real_time_enforcer/nat.tf [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+resource "google_compute_router" "main" {
+  name    = "${random_pet.main.id}"
+  network = "default"
+
+  bgp {
+    asn = "64514"
+  }
+
+  region  = "us-central1"
```
Comment (aaron-lane) [inline examples/real_time_enforcer/nat.tf]: We need to use a different region; this is causing a conflict with the `simple_example` tests.

**Comment by ocervell** [comment]
@aaron-lane changed the default to "false".

File: .kitchen.yml [hunk 2]
```diff
@@ -150,10 +150,11 @@ suites:
           backend: ssh
           controls:
             - real-time-enforcer-host
-          hosts_output: forseti-rt-enforcer-vm-public-ip
+          hosts_output: forseti-rt-enforcer-vm-ip
           user: ubuntu
           key_files:
             - test/fixtures/real_time_enforcer/sshkey
+          proxy_command: ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o IdentitiesOnly=yes -o LogLevel=ERROR -o ForwardAgent=no -i test/fixtures/bastion/tls_private_key ubuntu@<%= `terraform output -state test/fixtures/real_time_enforcer/terraform.tfstate.d/kitchen-terraform-real-time-enforcer-local/terraform.tfstate bastion_host`.strip %> -p 22 -W %h:%p
```
Comment (aaron-lane) [inline .kitchen.yml]: The `bastion_host` output must be added to `test/fixtures/real_time_enforcer/outputs.tf`

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,6 +15,7 @@ Extending the adopted spec, each change should have a link to its corresponding
 
 ### Changed
 
+- Make Forseti Real-Time Enforcer VM have private IP by default. [#180]
```
Comment (aaron-lane) [inline CHANGELOG.md]: This entry must go under the `Unreleased` section which has been readded on the master branch. Please rebase and correct the placement.

File: .kitchen.yml (hunk 2 above)
Comment (ocervell) [reply .kitchen.yml]: Done

**Comment by ocervell** [comment]
> Two more minor corrections to be made. Has a migration been tested to determine if these changes are non-breaking, even with the backward compatible default values?

Yes, a migration would work by just adding the `access_config` block to the resource and update the VM (no need to recreate it).

**Pull request merged**